### PR TITLE
Yu Miao & Joanna Philips - Music Releases

### PR DESCRIPTION
**Pull request by Joanna-Philips**
Deployed site: https://music-releases-yu-joanna.netlify.app/

### CODE REVIEW

File: code/src/components/Dropup.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import data from '../stretch-goal.json'
+
+const { useState } = React;
```
Comment (jenster5) [inline code/src/components/Dropup.js:4]: impressive you guys did all the stretch goals so cool!

**Review by jenster5 — COMMENTED** [review]
Wow! your project is great and you did all the stretch goals well done! The code is very clean and easy to read you used many file folders to keep everything neatly organized. Overall just a really well done project.

File: code/src/components/Dropup.js (hunk 1 above)
Comment (EvaSjoholm) [reply code/src/components/Dropup.js:4]: I agree with Jennie on this, really nice work!

File: code/src/components/Album.js
Comment (EvaSjoholm) [inline code/src/components/Album.js:1]: One Idea could be to use App.js for this section that you've implemented in Album.js, due to also having "Albums.js", which can get a bit confusing (very minor detail). But for separating the app part, and components added. :)

File: code/src/components/Albums.js
Comment (EvaSjoholm) [inline code/src/components/Albums.js:1]: My App.js, Album.js and Albums.js comment is referring then to this one. Minor details but just a thought.

**Review by EvaSjoholm — COMMENTED** [review]
Cool that you did stretch-goal and inspiring to see the code. Well done with the design too. Only input would be to use App.js in this specifik so the names for your components are bit clearer for the reader (album.js, albums.js) etc. Well done!

**Pull request closed without merging**